helm/charts/determined/templates/master-config.yaml

```diff
@@ -131,8 +131,8 @@ data:
       {{- toYaml .Values.resourcePools | nindent 6}}
     {{- end }}
 
-    {{$cpuImage := (split "/" "determinedai/environments:py-3.8-pytorch-1.12-tf-2.8-cpu-6218891")._1}}
```

Comment:
Should `change-password.yaml` also be updated?

Reply:
Oops, I missed that one (I think I only looked in `harness`). I don't know what that configuration is for, beyond the obvious (to change a password :-p), so I dunno how Tensorflow has anything to do with it.
All of these changes to image names are going to be reverted before I do the merge anyway (at least some will be properly updated with a bumpenv later).